bin/options.js

```diff
@@ -23,6 +23,7 @@ function getOptions () {
 
   try {
     var opts = fs.readFileSync(optsPath, 'utf8')
+      .replace(/^\s*#.*/gm, '')
```

Comment:
This could be changed to:
```js
    .replace((/^(\S*)\s*#.*$/m, '$1')`
```
to better accommodate comments at the end of a line, like so:
```
--recursive # To dive to deeper depths
```